tests/test_modules.py

```diff
@@ -58,3 +58,12 @@ def test_modules_install_fastqc_twice(self):
         """ Test installing a module - FastQC already there """
         self.mods.install("fastqc")
         assert self.mods.install("fastqc") is False
+
+    def test_modules_remove_fastqc(self):
+        """ Test removing FastQC module after installing it"""
+        self.mods.install("fastqc")
+        assert self.mods.remove("fastqc")
```

Comment:
Might be nice to assert that the files are actually gone too 😄

Reply:
👀 
Good point right there 😄